proxy/src/serverless/local_conn_pool.rs

```diff
@@ -39,7 +39,7 @@ use crate::usage_metrics::{Ids, MetricCounter, USAGE_METRICS};
 use crate::{DbName, RoleName};
 
 pub(crate) const EXT_NAME: &str = "pg_session_jwt";
-pub(crate) const EXT_VERSION: &str = "0.1.1";
+pub(crate) const EXT_VERSION: &str = "0.1.2";
```

Comment:
Is this something that we could just parse out of the file?

Reply:
That's an interesting idea. I'll have a think about that

Reply:
Cool. Doesn't have to block the PR.